bridge/contracts/ValidatorPool.sol

```diff
@@ -116,14 +112,13 @@ contract ValidatorPool is
     }
 
     function pauseConsensusOnArbitraryHeight(uint256 aliceNetHeight_) public onlyFactory {
-        require(
-            block.number >
-                ISnapshots(_snapshotsAddress()).getCommittedHeightFromLatestSnapshot() +
-                    MAX_INTERVAL_WITHOUT_SNAPSHOTS,
-            string(
-                abi.encodePacked(ValidatorPoolErrorCodes.VALIDATORPOOL_MIN_BLOCK_INTERVAL_NOT_MET)
-            )
-        );
+        if (
+            block.number <=
+            ISnapshots(_snapshotsAddress()).getCommittedHeightFromLatestSnapshot() +
+                MAX_INTERVAL_WITHOUT_SNAPSHOTS
+        ) {
+            revert ValidatorPoolErrors.MinimumBlockIntervalNotMet();
```

Comment:
can you add .getCommittedHeightFromLatestSnapshot() + MAX_INTERVAL_WITHOUT_SNAPSHOTS, and block.number to the variables in the error. for debugging purposes